### PR TITLE
Asset/item sprites closes #150

### LINKED ISSUE #150: Add new clothes and weapon sprite assets
**User Story**
As a game designer, I'd like to add new starting weapons, so that there is more variety in character builds and the bonuses the items provide.

**Acceptance Criteria**
Sprites for the following items are made:
- Slingshot 
- Longbow
- Robes
- Spellbook

**Notes**

---

**Why is this feature needed? Please describe the problem your requested feature wants to solve**

New items can't be added unless there is a pre-existing sprite to display them with

**Describe the solution you'd like**

The assets png files are created along with their respective index files detailing their stats.

### PR DESCRIPTION
**Pull request by KimberleyEvans-Parker**
GitHub Issue (If applicable): #150 

## PR Type

What kind of change does this PR introduce?
- Feature

## What is the current behavior?

Limited assets/items

## What is the new behavior?

New items: 
 - 5 robes
 - 5 staffs
 - 4 potions (2 health, 2 mana)
 - 4 ranged weapons (boomerang, slingshot, bow, crossbow)
 - 1 spell book (image only)

Robes and staffs are available from their allocated chest tier. 
All items apart from the spell book are able to be added to the shop (and chest for non-potions) when functionality is added to them in future PRs

Equipping robes and staves increases mana.
Mana potion functionality is left for #175 
All new weapons currently do melee damage only

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current upstream repo
- [ ] Added new tests to the test suite for the feature
- [x] Tested the full test suite, and ensured that the feature does not break current build.
- [ ] Docs have been added/updated which fit (for bug fixes / features)
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal)
- [ ] Checked licensing of Frameworks/Libraries (if applicable) 

## Other information

Internal Issue (If applicable):

closes #150

### CODE REVIEW

**Review by Matteas-Eden — CHANGES_REQUESTED** [review]
The game crashes on start with `TypeError: payload is undefined`. This only appears to affect the beginning of the game, and if I switch to the `master` branch and then switch back to your branch to circumvent the error, it doesn't seem to appear again.

**Review by Matteas-Eden — CHANGES_REQUESTED** [review]
I think this would satisfy the "Additional Items in Shop" section in the table in the README, so I'd like to see that updated.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
 [![All Contributors](https://img.shields.io/badge/all_contributors-4-orange.svg?style=flat-square)](#contributors-)
+
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
+
 #### SOFTENG 750/COMPSCI 732
 
-*Roll For Reaction* is an open-source fork of [react-rpg](https://github.com/ASteinheiser/react-rpg.com) created for SOFTENG750/COMPSCI 732 at the University of Auckland. This project aims to enhance the systems already present in the base game by modifying and adding elements to lightly emulate the popular tabletop role-playing game *Dungeons and Dragons: 5th Edition*. In doing so, the following features are planned to be added:
+_Roll For Reaction_ is an open-source fork of [react-rpg](https://github.com/ASteinheiser/react-rpg.com) created for SOFTENG750/COMPSCI 732 at the University of Auckland. This project aims to enhance the systems already present in the base game by modifying and adding elements to lightly emulate the popular tabletop role-playing game _Dungeons and Dragons: 5th Edition_. In doing so, the following features are planned to be added:
```
Comment (Matteas-Eden) [inline README.md:13]: These changes are unnecessary. Any particular reason they were made?

File: README.md [hunk 2]
```diff
@@ -15,7 +19,7 @@
 | ![cross][checkmark] Dice-based Combat             | ![check][checkmark] Save/Load System                  | ![cross][cross] Cinematic Cutscenes |
 | ![cross][cross] New Maps/Levels                   | ![cross][cross] New Enemy Types                       | ![cross][cross] Online Multiplayer  |
 | ![check][checkmark] Character Creation            | ![cross][checkmark] Customisable Outfit Colour        | ![cross][cross] Party System        |
-| ![cross][cross] Additional Items in Shop          | ![cross][cross] Split Screen Local Multiplayer        |                                     |
+| ![cross][checkmark] Additional Items in Shop      | ![cross][cross] Split Screen Local Multiplayer        |                                     |
```
Comment (Matteas-Eden) [inline README.md]: Although it won't change how it looks, would mine changing it to `![check][checkmark]`? Also change it on the others if you do.

**Review by Matteas-Eden — CHANGES_REQUESTED** [review]
Although it isn't the fault of this PR, there is a warning present due to an unused variable in `features/dialog-manager/dialogs/spellbook-dialog/index.js`. I'd appreciate it if you could remove that before this is merged. The other comments I've made here are non-blocking.

File: src/data/items/potions/mighty-mana-potion/index.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import MightyManaPotionImg from './mighty-mana-potion.png';
+
+const MightyManaPotion = {
+    name: 'Mana Potion',
```
Comment (Matteas-Eden) [inline src/data/items/potions/mighty-mana-potion/index.js]: The name here doesn't line up with the item name.

**Review by Matteas-Eden — CHANGES_REQUESTED** [review]
Sorry for not picking this up in my earlier review, but it seems that both of the mana potions are broken. When viewing the item, they claim to heal a value of `NAN` and when used they do exactly that, causing current health to become `NAN`.

File: README.md (hunk 1 above)
Comment (KimberleyEvans-Parker) [reply README.md:13]: They get changed when I hit save, so I can't really avoid it.  I assume it's a prettier thing.

**Comment by KimberleyEvans-Parker** [comment]
> Sorry for not picking this up in my earlier review, but it seems that both of the mana potions are broken. When viewing the item, they claim to heal a value of NAN and when used they do exactly that, causing current health to become NAN.

I've removed them from gameplay (like the ranged weapons).  Josh's PR #175 should bring them back in.

**Comment by Matteas-Eden** [comment]
> 
> 
> > Sorry for not picking this up in my earlier review, but it seems that both of the mana potions are broken. When viewing the item, they claim to heal a value of NAN and when used they do exactly that, causing current health to become NAN.
> 
> I've removed them from gameplay (like the ranged weapons). Josh's PR #175 should bring them back in.

Understood, however by doing so you introduced more warnings.

**Review by Matteas-Eden — APPROVED** [review]
Thanks for fixing those warnings, LGTM

**Pull request merged**